src/main/java/jsr223/docker/compose/DockerComposeCommandCreator.java

```diff
@@ -1,47 +1,41 @@
 package jsr223.docker.compose;
 
+import jsr223.docker.compose.utils.DockerComposePropertyLoader;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
 import java.util.ArrayList;
 import java.util.List;
 
-import jsr223.docker.compose.utils.DockerComposePropertyLoader;
-
+@NoArgsConstructor
 public class DockerComposeCommandCreator {
 
     // Constants
-    @lombok.Getter
-    private static String yamlFileName = "docker-compose.yml";
-    private static String filenameArgument = "-f";
-    private static String stopContainerArgument = "stop";
-    private static String removeContainerArgument = "rm";
-    private static String removeWithForceContainerArgument = "--force";
-    private static String setupContainerArgument = "up";
-
-
-    /**
-     * Construct the docker compose stop command.
-     *
-     * @return String array representing a command.
-     */
-    public static String[] createDockerComposeStopCommand() {
-        List<String> command = new ArrayList<>();
-        addSudoAndDockerComposeCommand(command);
+    @Getter
+    private final String yamlFileName = "docker-compose.yml";
+    @Getter
+    private final String filenameArgument = "-f";
+    @Getter
+    private final String setupContainerArgument = "up";
+    @Getter
+    private final String stopAndRemoveContainerArgument = "down";
+    @Getter
+    private final String volumesArgument = "--volumes";
```

Comment:
lombok @Data annotation on class level can be used instead @Getter on each field. if the field is final, only getter methods will be generated by lombok. This can save you some code.